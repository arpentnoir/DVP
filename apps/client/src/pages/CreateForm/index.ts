export * from './AANZFTACOOSchema';
export * from './CreateFormPage';
