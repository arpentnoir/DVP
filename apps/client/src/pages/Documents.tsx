import { DataTable, MoreInfo, Text } from '@dvp/vc-ui';
import { Box } from '@mui/material';
import { GridColDef } from '@mui/x-data-grid';
import { useNavigate } from 'react-router-dom';
import { BaseLayout } from '../layouts';

// TODO: Replace with real data
const columns: GridColDef[] = [
  {
    field: 'id',
    headerName: 'ID',
    minWidth: 200,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'ABN',
    headerName: 'ABN',
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'businessName',
    headerName: 'Business name',
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'document',
    headerName: 'Document',
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'dateIssued',
    headerName: 'Date Issued',
    sortable: false,
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'moreInfo',
    headerName: '',
    flex: 1,
    renderCell: (params) => (
      <MoreInfo
        params={params}
        items={{ View: () => null, Download: () => null, Revoke: () => null }}
      />
    ),
    disableColumnMenu: true,
    sortable: false,
  },
];

const rows = [
  {
    id: '5940107005302237590',
    document: 'AANZFTA',
    businessName: 'Organisation',
    ABN: '1234 5678 90',
    dateIssued: '01/01/2012',
  },
  {
    id: '5940107005302237591',
    document: 'CITES',
    businessName: 'Organisation',
    ABN: '1234 5678 91',
    dateIssued: '01/01/2012',
  },
  {
    id: '5940107005302237592',
    document: 'AANZFTA',
    businessName: 'Organisation',
    ABN: '1234 5678 92',
    dateIssued: '01/01/2018',
  },
  {
    id: '5940107005302237593',
    document: 'CITES',
    businessName: 'Organisation',
    ABN: '1234 5678 93',
    dateIssued: '01/01/2018',
  },
  {
    id: '5940107005302237594',
    document: 'CITES',
    businessName: 'Organisation',
    ABN: '1234 5678 94',
    dateIssued: '01/01/2032',
  },
  {
    id: '5940107005302237595',
    document: 'AANZFTA',
    businessName: 'Organisation',
    ABN: '1234 5678 95',
    dateIssued: '01/01/2019',
  },
  {
    id: '5940107005302237596',
    document: 'CITES',
    businessName: 'Organisation',
    ABN: '1234 5678 96',
    dateIssued: '01/01/2019',
  },
  {
    id: '5940107005302237597',
    document: 'CITES',
    businessName: 'Organisation',
    ABN: '1234 5678 97',
    dateIssued: '01/01/2020',
  },
  {
    id: '5940107005302237598',
    document: 'AANZFTA',
    businessName: 'Organisation',
    ABN: '1234 5678 98',
    dateIssued: '01/01/2022',
  },
];

export const Documents = () => {
  const navigate = useNavigate();

  const handleIssue = () => {
    navigate('/issue');
  };

  return (
    <BaseLayout title="Documents">
      <Box paddingBottom="40px">
        <Text variant="h1" fontWeight="bold" paddingBottom="24px">
          Documents
        </Text>
        <Text>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu
          turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec
          fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus
          elit sed risus. Maecenas eget condimentum velit, sit amet feugiat
          lectus. Class aptent taciti sociosqu ad litora torquent per conubia
          nostra, per inceptos himenaeos. Praesent auctor purus luctus enim
          egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex.
          Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum
          lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in
          elementum tellus.
        </Text>
        <Text variant="h3" fontWeight="bold" paddingTop="40px">
          Documents issued
        </Text>
        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Text>
      </Box>

      <DataTable
        handleAction={handleIssue}
        toolBarActionLabel="Issue"
        rows={rows}
        columns={columns}
        pageSize={5}
        rowsPerPageOptions={[5]}
        checkboxSelection
      />
    </BaseLayout>
  );
};
