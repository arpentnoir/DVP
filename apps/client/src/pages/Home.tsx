import { Card, Text } from '@dvp/vc-ui';
import NavigateNextIcon from '@mui/icons-material/NavigateNext';
import { Box, ButtonBase, Grid, Stack, useTheme } from '@mui/material';
import { useNavigate } from 'react-router-dom';
import { ROUTES } from '../constants';
import { BaseLayout } from '../layouts';

// TODO: Replace with real data
const actions = [
  {
    name: 'Lorem',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Ipsum',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Dolor',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
  {
    name: 'Amet',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
  },
];

export const Home = () => {
  const theme = useTheme();
  const navigate = useNavigate();

  return (
    <BaseLayout
      title="Digital Verification Platform"
      sx={{
        paddingX: 0,
        paddingY: 0,
        backgroundColor: theme.palette.common.backgroundBlue,
        display: 'flex',
        justifyContent: 'center',
        maxWidth: '100%',
      }}
    >
      <Grid container height={'100%'} maxWidth={'1140px'}>
        <Stack>
          <Stack
            direction={'row'}
            sx={{
              height: 'calc(50vw * 0.62)',
              minHeight: { xs: '370px', md: '410px' },
              maxHeight: '450px',
              width: '100%',
            }}
          >
            <Grid item xs={12} sm={6}>
              <Stack
                padding={{
                  xs: '60px 20px 0px 20px',
                  sm: '40px 30px',
                  md: '60px 0px 0px 30px',
                }}
              >
                <Box>
                  <Text variant="h1" fontWeight="bold" color="white">
                    DIGITAL VERIFICATION
                  </Text>
                  <Text variant="h1" fontWeight="bold" color="white">
                    PLATFORM
                  </Text>
                </Box>
                <Text color="white" paddingTop="12px">
                  Lorem ipsum dolar sit amet consectetur
                </Text>
                <Stack padding="8px 10px 0 0">
                  <ButtonBase onClick={() => navigate(ROUTES.VERIFY)}>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Verify a document
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                  <ButtonBase>
                    <Stack
                      width="100%"
                      direction="row"
                      justifyContent="space-between"
                      alignItems="center"
                      sx={{
                        borderBottom: '1px solid white',
                        padding: '16px 20px',
                      }}
                    >
                      <Text color="white" fontWeight="bold" variant="body1">
                        Help and support
                      </Text>
                      <NavigateNextIcon
                        sx={{ color: theme.palette.common.featureGold }}
                      />
                    </Stack>
                  </ButtonBase>
                </Stack>
              </Stack>
            </Grid>
            <Grid
              item
              sm={6}
              sx={{
                display: {
                  xs: 'none',
                  sm: 'block',
                  background: 'url("assets/home-background.jpg")',
                  backgroundPosition: 'bottom right',
                  backgroundSize: 'cover',
                  backgroundRepeat: 'no-repeat',
                  overflow: 'hidden',
                },
                position: 'absolute',
                width: '100%',
                right: 0,
                height: 'calc(50vw * 0.62)',
                minHeight: { xs: '370px', md: '410px' },
                maxHeight: '450px',
              }}
            >
              <Box
                sx={{
                  background: theme.palette.common.backgroundBlue,
                  position: 'relative',
                  top: '-100px',
                  left: '-120px',
                  width: '200px',
                  height: 'calc(100% + 300px)',
                  transform: 'rotate(-15deg)',
                }}
              />
            </Grid>
          </Stack>
          <Grid
            container
            spacing="30px"
            padding={{
              xs: '0px 40px 100px 40px',
              sm: '60px 40px',
            }}
          >
            {actions.map((action) => (
              <Grid item lg={3} md={4} sm={6} xs={12} key={action.name}>
                <Card
                  sx={{ flex: 1, padding: '16px' }}
                  name={action.name}
                  text={action.text}
                  handleAction={() => null}
                  actionLabel="Action"
                />
              </Grid>
            ))}
          </Grid>
        </Stack>
      </Grid>
    </BaseLayout>
  );
};
