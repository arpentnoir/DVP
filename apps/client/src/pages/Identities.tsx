import { Button, Card, DataTable, MoreInfo, Text } from '@dvp/vc-ui';
import AddIcon from '@mui/icons-material/Add';
import { Box, Stack } from '@mui/material';
import { GridColDef } from '@mui/x-data-grid';
import { BaseLayout } from '../layouts';

// TODO: Replace with real data
const columns: GridColDef[] = [
  {
    field: 'nickname',
    headerName: 'Nickname',
    minWidth: 200,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'id',
    headerName: 'Public key',
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'dateIssued',
    headerName: 'Date Issued',
    sortable: false,
    minWidth: 150,
    flex: 1,
    disableColumnMenu: true,
  },
  {
    field: 'moreInfo',
    headerName: '',
    flex: 1,
    renderCell: (params) => (
      <MoreInfo
        params={params}
        items={{ Remove: () => null, Revoke: () => null }}
      />
    ),
    disableColumnMenu: true,
    sortable: false,
  },
];

const rows = [
  {
    id: '5940107005302237590',
    nickname: 'ABC Internal',
    dateIssued: '01/01/2012',
  },
  {
    id: '5940107005302237591',
    nickname: 'ABC External',
    dateIssued: '01/01/2012',
  },
  {
    id: '5940107005302237592',
    nickname: 'XYZ Private',
    dateIssued: '01/01/2018',
  },
  {
    id: '5940107005302237593',
    nickname: 'XYZ Public',
    dateIssued: '01/01/2018',
  },
];

const identities = [
  {
    name: 'Identity 1',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
    handleAction: () => null,
  },
  {
    name: 'Identity 2',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
    handleAction: () => null,
  },
  {
    name: 'Identity 3',
    text: 'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.',
    handleAction: () => null,
  },
];

export const Identities = () => {
  const handleCreateNew = () => null;

  return (
    <BaseLayout title="Identities">
      <Box paddingBottom="40px">
        <Text variant="h4" fontWeight="bold" paddingBottom="24px">
          Identities
        </Text>
        <Text>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu
          turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec
          fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus
          elit sed risus. Maecenas eget condimentum velit, sit amet feugiat
          lectus. Class aptent taciti sociosqu ad litora torquent per conubia
          nostra, per inceptos himenaeos. Praesent auctor purus luctus enim
          egestas, ac scelerisque ante pulvinar. Donec ut rhoncus ex.
          Suspendisse ac rhoncus nisl, eu tempor urna. Curabitur vel bibendum
          lorem. Morbi convallis convallis diam sit amet lacinia. Aliquam in
          elementum tellus.
        </Text>
      </Box>

      <Box paddingBottom="40px">
        <Text variant="body1" fontWeight="bold">
          Identifiers (DIDs)
        </Text>
        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Text>
        <Stack
          direction={{ xs: 'column', sm: 'row' }}
          spacing="8px"
          paddingY="40px"
        >
          {identities.map((identity) => (
            <Card
              key={identity.name}
              name={identity.name}
              text={identity.text}
              handleAction={identity.handleAction}
              actionLabel="Remove"
            />
          ))}
        </Stack>
        <Stack direction="row" spacing="16px">
          <Button label="Upload DID" variant="outlined" />
          <Button label="Create new" leftIcon={<AddIcon />} color="error" />
        </Stack>
      </Box>

      <Box paddingBottom="30px">
        <Text variant="body1" fontWeight="bold">
          Key pairs
        </Text>
        <Text>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</Text>
      </Box>

      <DataTable
        handleAction={handleCreateNew}
        toolBarActionLabel="Create new"
        rows={rows}
        columns={columns}
        pageSize={5}
        rowsPerPageOptions={[5]}
        checkboxSelection
      />
    </BaseLayout>
  );
};
