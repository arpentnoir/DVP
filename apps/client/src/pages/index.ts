export * from './CreateForm';
export * from './Home';
export * from './Verify';
export * from './Viewer';
export * from './Issue';
