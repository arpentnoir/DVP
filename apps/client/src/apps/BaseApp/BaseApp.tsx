import React from 'react';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
import { CssBaseline, ThemeProvider, Container } from '@mui/material';
import { NavBar } from '../../components';
import { CreateFormPage, Home, Verify, Viewer, Issue } from '../../pages';
import { theme } from '../../theme';

interface IBaseApp {
  children?: React.ReactNode;
}

export const BaseApp: React.FC<IBaseApp> = ({ children }) => {
  return (
    <React.Fragment>
      <CssBaseline />
      <ThemeProvider theme={theme}>
        <Router>
          <Container>
            <NavBar />
            <Routes>
              <Route
                path="/form"
                element={
                  <CreateFormPage
                    formTitle={'Create AANZFTA Certificate of Origin'}
                  />
                }
              />
              <Route path="/" element={<Home />} />
              <Route path="/verify" element={<Verify />} />
              <Route path="/viewer" element={<Viewer />} />
              <Route path="/issue" element={<Issue />} />

              {/* Additional routes for Internet and Internal Apps */}
              {children}
            </Routes>
          </Container>
        </Router>
      </ThemeProvider>
    </React.Fragment>
  );
};
