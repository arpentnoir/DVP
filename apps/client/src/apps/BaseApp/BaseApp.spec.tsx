import { getByText, render } from '@testing-library/react';
import { useEffect } from 'react';
import { useNavigate } from 'react-router-dom';
import * as pages from '../../pages';
import { BaseAppWithoutAuth as BaseApp } from './BaseApp';

// eslint-disable-next-line @typescript-eslint/no-unsafe-return
jest.mock('../../pages', () => ({
  ...jest.requireActual('../../pages'),
}));

describe('BaseApp', () => {
  it('should render correctly', () => {
    const { baseElement } = render(<BaseApp />);

    expect(baseElement).toMatchSnapshot();
  });

  it('should display app error component if app throws uncaught error', () => {
    (jest.spyOn(pages, 'Home') as jest.SpyInstance).mockImplementation(() => {
      throw Error('Soopa Doopa');
    });

    const { baseElement } = render(<BaseApp />);

    getByText(baseElement, 'Something went wrong');
  });

  it('should display not found error component if route is not defined', () => {
    (jest.spyOn(pages, 'Home') as jest.SpyInstance).mockImplementation(() => {
      const navigate = useNavigate();
      useEffect(() => {
        navigate('nonExistentPage');
      }, []);
    });

    const { baseElement } = render(<BaseApp />);

    getByText(baseElement, 'PAGE NOT FOUND');
  });
});
