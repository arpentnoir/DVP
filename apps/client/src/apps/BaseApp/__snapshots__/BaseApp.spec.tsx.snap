// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseApp should render correctly 1`] = `
<body>
  <div>
    <div
      class="css-1rzuxdl-MuiStack-root"
    >
      <header
        class="MuiPaper-root MuiPaper-elevation MuiPaper-elevation0 MuiAppBar-root MuiAppBar-colorPrimary MuiAppBar-positionStatic css-lk4nos-MuiPaper-root-MuiAppBar-root"
      >
        <div
          class="css-1dlss83-MuiStack-root"
        >
          <div
            class="MuiBox-root css-hybe8"
          />
          <div
            class="css-uljwwc-MuiStack-root"
          >
            <img
              alt="Australian Border Force Logo"
              class="MuiBox-root css-1uzisz9"
              src="assets/logo.svg"
              tabindex="0"
            />
          </div>
          <div
            class="MuiBox-root css-lt80mk"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-eytzub-MuiButtonBase-root-MuiButton-root"
              data-testid="button:LOGIN"
              tabindex="0"
              type="button"
            >
              <div
                class="css-doi3ij-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-f9se0d-MuiTypography-root"
                  data-testid="button:LOGIN:label"
                  tabindex="0"
                >
                  LOGIN
                </p>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </header>
      <div
        class="MuiContainer-root MuiContainer-disableGutters css-dpgzcu-MuiContainer-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container css-uv4xu-MuiGrid-root"
        >
          <div
            class="css-nen11g-MuiStack-root"
          >
            <div
              class="css-7zlskd-MuiStack-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-1b3l6lk-MuiGrid-root"
              >
                <div
                  class="css-hwbtf3-MuiStack-root"
                >
                  <div
                    class="MuiBox-root css-0"
                  >
                    <h1
                      class="MuiTypography-root MuiTypography-h1 css-a226le-MuiTypography-root"
                      tabindex="0"
                    >
                      DIGITAL VERIFICATION
                    </h1>
                    <h1
                      class="MuiTypography-root MuiTypography-h1 css-a226le-MuiTypography-root"
                      tabindex="0"
                    >
                      PLATFORM
                    </h1>
                  </div>
                  <p
                    class="MuiTypography-root MuiTypography-body2 css-1nuaw8y-MuiTypography-root"
                    tabindex="0"
                  >
                    Lorem ipsum dolar sit amet consectetur
                  </p>
                  <div
                    class="css-124fyjl-MuiStack-root"
                  >
                    <button
                      class="MuiButtonBase-root css-10d1a0h-MuiButtonBase-root"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-1lkzb43-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-fp1q5u-MuiTypography-root"
                          tabindex="0"
                        >
                          Verify a document
                        </p>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1vdifv0-MuiSvgIcon-root"
                          data-testid="NavigateNextIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                          />
                        </svg>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                    <button
                      class="MuiButtonBase-root css-10d1a0h-MuiButtonBase-root"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-1lkzb43-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body1 css-fp1q5u-MuiTypography-root"
                          tabindex="0"
                        >
                          Help and support
                        </p>
                        <svg
                          aria-hidden="true"
                          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1vdifv0-MuiSvgIcon-root"
                          data-testid="NavigateNextIcon"
                          focusable="false"
                          viewBox="0 0 24 24"
                        >
                          <path
                            d="M10 6 8.59 7.41 13.17 12l-4.58 4.59L10 18l6-6z"
                          />
                        </svg>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-sm-6 css-w5l21d-MuiGrid-root"
              >
                <div
                  class="MuiBox-root css-jnllwp"
                />
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-container css-1mru3pf-MuiGrid-root"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-3 css-14j1sok-MuiGrid-root"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1veos4r-MuiPaper-root-MuiCard-root"
                >
                  <div
                    class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
                  >
                    <div
                      class="css-3y1137-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-j63srw-MuiTypography-root"
                        tabindex="0"
                      >
                        Lorem
                      </p>
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-gjhwj6-MuiTypography-root"
                      tabindex="0"
                    >
                      It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                    </p>
                  </div>
                  <div
                    class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium css-c09va-MuiButtonBase-root-MuiButton-root"
                      data-testid="button:Action"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-doi3ij-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 css-13sysnx-MuiTypography-root"
                          data-testid="button:Action:label"
                          tabindex="0"
                        >
                          Action
                        </p>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-3 css-14j1sok-MuiGrid-root"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1veos4r-MuiPaper-root-MuiCard-root"
                >
                  <div
                    class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
                  >
                    <div
                      class="css-3y1137-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-j63srw-MuiTypography-root"
                        tabindex="0"
                      >
                        Ipsum
                      </p>
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-gjhwj6-MuiTypography-root"
                      tabindex="0"
                    >
                      It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                    </p>
                  </div>
                  <div
                    class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium css-c09va-MuiButtonBase-root-MuiButton-root"
                      data-testid="button:Action"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-doi3ij-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 css-13sysnx-MuiTypography-root"
                          data-testid="button:Action:label"
                          tabindex="0"
                        >
                          Action
                        </p>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-3 css-14j1sok-MuiGrid-root"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1veos4r-MuiPaper-root-MuiCard-root"
                >
                  <div
                    class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
                  >
                    <div
                      class="css-3y1137-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-j63srw-MuiTypography-root"
                        tabindex="0"
                      >
                        Dolor
                      </p>
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-gjhwj6-MuiTypography-root"
                      tabindex="0"
                    >
                      It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                    </p>
                  </div>
                  <div
                    class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium css-c09va-MuiButtonBase-root-MuiButton-root"
                      data-testid="button:Action"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-doi3ij-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 css-13sysnx-MuiTypography-root"
                          data-testid="button:Action:label"
                          tabindex="0"
                        >
                          Action
                        </p>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 MuiGrid-grid-md-4 MuiGrid-grid-lg-3 css-14j1sok-MuiGrid-root"
              >
                <div
                  class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1veos4r-MuiPaper-root-MuiCard-root"
                >
                  <div
                    class="MuiCardContent-root css-46bh2p-MuiCardContent-root"
                  >
                    <div
                      class="css-3y1137-MuiStack-root"
                    >
                      <p
                        class="MuiTypography-root MuiTypography-body1 css-j63srw-MuiTypography-root"
                        tabindex="0"
                      >
                        Amet
                      </p>
                    </div>
                    <p
                      class="MuiTypography-root MuiTypography-body2 css-gjhwj6-MuiTypography-root"
                      tabindex="0"
                    >
                      It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout.
                    </p>
                  </div>
                  <div
                    class="MuiCardActions-root MuiCardActions-spacing css-1t6e9jv-MuiCardActions-root"
                  >
                    <button
                      class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textError MuiButton-sizeMedium MuiButton-textSizeMedium css-c09va-MuiButtonBase-root-MuiButton-root"
                      data-testid="button:Action"
                      tabindex="0"
                      type="button"
                    >
                      <div
                        class="css-doi3ij-MuiStack-root"
                      >
                        <p
                          class="MuiTypography-root MuiTypography-body2 css-13sysnx-MuiTypography-root"
                          data-testid="button:Action:label"
                          tabindex="0"
                        >
                          Action
                        </p>
                      </div>
                      <span
                        class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                      />
                    </button>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="css-1sei285-MuiStack-root"
      >
        <div
          class="css-1qqqzbk-MuiStack-root"
        >
          <div
            class="css-hyap7f-MuiStack-root"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-eytzub-MuiButtonBase-root-MuiButton-root"
              data-testid="button:Who we are"
              tabindex="0"
              type="button"
            >
              <div
                class="css-doi3ij-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1213u1-MuiTypography-root"
                  data-testid="button:Who we are:label"
                  tabindex="0"
                >
                  Who we are
                </p>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-eytzub-MuiButtonBase-root-MuiButton-root"
              data-testid="button:Our Ministers"
              tabindex="0"
              type="button"
            >
              <div
                class="css-doi3ij-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1213u1-MuiTypography-root"
                  data-testid="button:Our Ministers:label"
                  tabindex="0"
                >
                  Our Ministers
                </p>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-eytzub-MuiButtonBase-root-MuiButton-root"
              data-testid="button:About us"
              tabindex="0"
              type="button"
            >
              <div
                class="css-doi3ij-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1213u1-MuiTypography-root"
                  data-testid="button:About us:label"
                  tabindex="0"
                >
                  About us
                </p>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-eytzub-MuiButtonBase-root-MuiButton-root"
              data-testid="button:Contact us"
              tabindex="0"
              type="button"
            >
              <div
                class="css-doi3ij-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body2 css-1213u1-MuiTypography-root"
                  data-testid="button:Contact us:label"
                  tabindex="0"
                >
                  Contact us
                </p>
              </div>
              <span
                class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
              />
            </button>
          </div>
          <img
            alt="Australian Government"
            class="MuiBox-root css-1nbic85"
            src="assets/footer-logo-white.svg"
            tabindex="0"
          />
        </div>
      </div>
    </div>
  </div>
</body>
`;
